dist/SpreadTheWord.d.ts

```diff
@@ -6,7 +6,8 @@ import Listener, { ListenerOptions } from "./Listener";
 export declare type StatusType = "uninitialized" | "spreaded" | "destroyed";
 export default class SpreadTheWord extends EventEmitter {
     server: Server;
-    services: Service[];
```

Comment:
Couldn't use `eventers` here, so renamed both to `servicesList` and `listenersList`.